gwcs/selector.py

```diff
@@ -140,70 +375,35 @@ class RegionsSelector(Model):
         Names of the outputs.
     selector : dict
         Mapping of region labels to transforms.
-        Labels can be of type int or str, transforms are of type `~astropy.modeling.core.Model`
-    mask : `~gwcs.selector.SelectorMask`
-        Mask with region labels.
+        Labels can be of type int or str, transforms are of type `~astropy.modeling.core.Model`.
+    label_mapper : a subclass of `~gwcs.selector._LabelMapper`
+        A model which maps locations to region labels.
     undefined_transform_value : float, np.nan (default)
         Value to be returned if there's no transform defined for the inputs.
     """
+    standard_broadcasting = False
```

Comment:
The `_param_names` below doesn't seem to be used anywhere (or was it used to work around that previous bug with parameter-less models?)  It can probably go away now?